src/main/com/algos/src/maths/AbsoluteMax.java

```diff
@@ -13,12 +13,12 @@ public static int getMaxValue(int... numbers) {
         if (numbers == null || numbers.length == 0) {
             throw new IllegalArgumentException("Numbers array cannot be empty or null");
         }
-        int absMax = numbers[0];
+        int absMax = 0;  
         for (int i = 1; i < numbers.length; i++) {
-            if (Math.abs(numbers[i]) > Math.abs(absMax)) {
+            if (numbers[i] > absMax) {
```

Comment:
**Issue**: The condition in the loop does not correctly compute the maximum absolute value, as it compares the numbers directly rather than their absolute values. <br> **Fix**: Modify the condition to compare the absolute values of the numbers. <br> **Code Suggestion**: 
 ```
 18c18
 <             if (numbers[i] > absMax) {
 ---
 >             if (Math.abs(numbers[i]) > Math.abs(absMax)) {
 ```